--- eBot/Config/Config.php.org	2016-03-13 01:38:12.364640966 +0100
+++ eBot/Config/Config.php	2016-03-13 03:06:07.452640966 +0100
@@ -43,7 +43,7 @@
     public function __construct() {
         $options = getopt("", array("config::"));  
         $file = APP_ROOT . DIRECTORY_SEPARATOR . "config" . DIRECTORY_SEPARATOR . "config.ini";
-        if (@$options['config']) {
+        if (isset($options['config'])) {
             if (file_exists(APP_ROOT . DIRECTORY_SEPARATOR . "config" . DIRECTORY_SEPARATOR . $options['config'])) {
                 $file = APP_ROOT . DIRECTORY_SEPARATOR . "config" . DIRECTORY_SEPARATOR . $options['config'];
             }
