--- Match.php.orig      2016-10-29 20:14:25.576877906 +0000
+++ Match.php   2016-10-29 20:15:54.859505653 +0000
@@ -784,7 +784,7 @@
         }
     }

-    public function say($message, $color) {
+    public function say($message, $color = 0) {
         // Want whole line in a certain color?
         if ( $color )
             $message = $this->formatText($message, $color);
@@ -806,7 +806,7 @@
         }
     }

-    public function say_player($playerId, $message, $color) {
+    public function say_player($playerId, $message, $color = 0) {
         if ($color)
             $message = $this->formatText($message, $color);

