--- Match.php.org	2016-03-16 21:22:19.477246266 +0100
+++ /home/ebotv3/eBot-CSGO/src/eBot/Match/Match.php	2016-03-16 21:28:19.857246266 +0100
@@ -1364,6 +1364,8 @@
                 $this->rcon->send("mp_do_warmup_period 1");
                 $this->rcon->send("mp_warmuptime 30");
                 $this->rcon->send("mp_warmup_pausetimer 1");
+                $this->rcon->send("mp_ct_default_secondary \"weapon_hkp2000\"");
+                $this->rcon->send("mp_t_default_secondary \"weapon_glock\"");
                 $this->rcon->send("mp_warmup_start");
                 $this->say("nothing change, going to warmup");
             }
@@ -1379,6 +1381,8 @@
                 $this->rcon->send("mp_do_warmup_period 1");
                 $this->rcon->send("mp_warmuptime 30");
                 $this->rcon->send("mp_warmup_pausetimer 1");
+                $this->rcon->send("mp_ct_default_secondary \"weapon_hkp2000\"");
+                $this->rcon->send("mp_t_default_secondary \"weapon_glock\"");
                 $this->rcon->send("mp_warmup_start");
                 $this->say("Swapping teams");
                 $this->rcon->send("mp_swapteams");
@@ -2714,7 +2718,7 @@
 
                     $this->rcon->send("exec " . $this->matchData["rules"] . ".cfg; mp_warmuptime 0; mp_halftime_pausetimer 1; mp_warmup_pausetimer 0;");
                     $this->rcon->send("sv_rcon_whitelist_address \"" . \eBot\Config\Config::getInstance()->getBot_ip() . "\"");
-                    $this->rcon->send("mp_halftime_duration 1; mp_roundtime_defuse 60");
+                    $this->rcon->send("mp_halftime_duration 1; mp_roundtime_defuse 60; mp_ct_default_secondary \"\"; mp_t_default_secondary \"\"; mp_startmoney 0;");
                     $this->rcon->send("mp_warmup_end");
                     if (\eBot\Config\Config::getInstance()->getKo3Method() == "csay" && $this->pluginCsay) {
                         $this->rcon->send("csay_ko3");
@@ -2750,7 +2754,7 @@
                             // NEW
                             $this->rcon->send("exec $fichier; mp_warmuptime 0; mp_halftime_pausetimer 1;");
                             $this->rcon->send("sv_rcon_whitelist_address \"" . \eBot\Config\Config::getInstance()->getBot_ip() . "\"");
-                            $this->rcon->send("mp_halftime_duration 1");
+                            $this->rcon->send("mp_halftime_duration 1; mp_ct_default_secondary \"weapon_hkp2000\"; mp_t_default_secondary \"weapon_glock\";");
                             $this->rcon->send("mp_warmup_end");
                             if (\eBot\Config\Config::getInstance()->getLo3Method() == "csay" && $this->pluginCsay) {
                                 $this->rcon->send("csay_lo3");
